themes/V3/5ePHB/style.less

```diff
@@ -677,6 +678,136 @@ body {
 		margin-top : 0.2cm;
 	}
 }
+//*****************************
+// *      COVER PAGE
+// *****************************/
+
+.page {
+	&:has(.coverPage):after {
+		display: none;
+	}
+	.coverPage {
+		position: absolute;
+		inset: 0;
+		padding: 125px 0px;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		text-align: center;
+
+		p {
+			color: white;
+			font-size: 20px;
+		}
+		> p > img {
+			position: absolute;
+			inset: 0;
+			height: 100%;
+			min-width: 100%;
+		}
+		&:before {
+			background-image: @homebreweryLogo;
+			background-position: center;
+			background-size: 75px;
```

Comment:
In general, let's try to set absolute sizes using cm instead of px. We haven't been perfect about this, but the basic rule we've been following is:

- **cm** for sizes related to the page (width of an image, font size, padding, etc.)
- **px** for sizes related to CSS filters and specific pixel cutoffs (borders, shadows)

In the same vein, we try to avoid `em` for setting font sizes, and instead use `cm`. EM is ok for line heights (i.e., when a line is 1.2 times the height of the font, use `em 1.2`), but we've run into a lot of issues in the past where we change one font size and the whole document breaks.